frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/ResetInfoDialog.kt

```diff
@@ -0,0 +1,37 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.ide.util.PropertiesComponent
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.openapi.ui.Messages
+import com.intellij.ui.dsl.builder.panel
+import com.virtuslab.gitmachete.frontend.actions.base.BaseResetToRemoteAction
+import com.virtuslab.gitmachete.frontend.actions.compat.rowCompat
+import com.virtuslab.gitmachete.frontend.resourcebundles.GitMacheteBundle.getString
+import org.checkerframework.checker.tainting.qual.Tainted
+import java.awt.Dimension
+
+class ResetInfoDialog(project: Project, private val content: @Tainted String) :
+  DialogWrapper(project, /* canBeParent */ true) {
+
+  init {
+    title = getString("action.GitMachete.BaseResetToRemoteAction.info-dialog.title")
+    setOKButtonMnemonic('O'.code)
+    setDoNotAskOption(object : com.intellij.openapi.ui.DoNotAskOption.Adapter() {
```

Comment:
Actually, _one simple trick_ to avoid instantiating an inner class in c'tor here: create a private method `createDoNotAskOptionAdapter(project: Project)` that will return this very `object`... and then

```suggestion
    setDoNotAskOption(createDoNotAskOptionAdapter(project))
```

Even better, `createDoNotAskOptionAdapter` can be a "static" method (so in Kotlin terms, defined in companion object).